app/lib/services/funding_eligibility.rb

```diff
@@ -35,9 +48,13 @@ def funded?
 
     def funding_eligiblity_status_code
       @funding_eligiblity_status_code ||= begin
+        if approved_itt_provider && lead_mentor
+          return lead_mentor_eligibility_status
+        end
```

Comment:
Have spoken to Sam and understand that the approved_itt_provider being false if you're a lead mentor is an edge-case that shouldn't occur since proceeding past that point in the journey would require you to force your way past. These users receiving a NO_INSTITUTION error is correct since they *should* have an institution (an approved_itt_provider) but don't.